go/app/infra.go

```diff
@@ -32,120 +28,197 @@ type jsonData struct {
 type ItemRepository interface {
 	Insert(ctx context.Context, item *Item) error
 	GetAll(ctx context.Context) ([]Item, error)
-	GetByID(ctx context.Context, id int) (*Item, error)
+	GetItemByID(ctx context.Context, id int) (*Item, error)
+	Getkeyword(ctx context.Context, keyword string) ([]Item, error)
```

Comment:
```suggestion
	GetKeyword(ctx context.Context, keyword string) ([]Item, error)
```

Golangでは命名規則の慣例があります。関数の場合、キャメルケースでの命名を行います。

この記事が参考になるかもしれません！
https://zenn.dev/keitakn/articles/go-naming-rules